relayer/src/__tests__/source.test.ts

```diff
@@ -0,0 +1,56 @@
+import { U64 } from "@polkadot/types/primitive";
+import { Observable } from "rxjs";
+import Source from "../source";
+import { apiMock, loggerMock } from "../mocks";
+
+describe("Source class", () => {
+    const params = {
+        api: apiMock,
+        chain: "Random relay chain name",
+        feedId: 66 as unknown as U64,
+        parachainsMap: new Map(),
+        logger: loggerMock,
+    };
+
+    it("should create an instance", () => {
+        const source = new Source(params);
+
+        expect(source).toBeInstanceOf(Source);
+        expect(source).toHaveProperty("subscribeBlocks");
```

Comment:
Hm, the fact that it has such property is not super useful, it should be a method, not a property (it is not a getter in the class either).